src/modules/roc_address/target_posix/roc_address/socket_addr_enums.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_address/target_posix/roc_address/socket_addr_enums.h
```

Comment:
This header is not posix-specific, so let's move it outside of the target directory. 

Also, IIRC AddrType will be used in roc_sdp even without SocketAddr; so maybe rename from socket_addr_enums.h to addr_type.h.